### ISSUE TITLE
Dashboard: Focus visible not at correct location (mobile)

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#31]
URL: https://hcdev.openplus.ca/en/user/203 (Dashboard)
User Agent: iPhone 12 (iOS 14.7.1)
Language: English

Issue Description: Focus on iOS doesn't seem to be at the right location for hidden content and goes up and down a little bit. This is  very confusing as there is a lot of hidden content on top of the pages that receives focus and the focus box appears at random areas on the screen.

Steps to reproduce: Turn VoiceOver on on iPhone. Listen to full page either by 2-finger swipe up OR by 1-finger swipe right item by item. Its

Expected results: The focus should be at the correct location. When some of the hidden content is made visible, this might fix some of these focus issues.

![image](https://user-images.githubusercontent.com/14280062/128202824-5ce2ec0c-5186-4c79-bb4b-638831e8437a.png)

![image](https://user-images.githubusercontent.com/14280062/128203927-ae6e4c3c-7043-4ac5-9d5e-9278b30c7b5c.png)

![image](https://user-images.githubusercontent.com/14280062/128201162-4d4e0c6e-be53-41b9-bcc1-ec584f0b786b.png)

**Comment by davidgotlieb** [comment]
Could not duplicate, may have been fixed with WxT update

**Comment by ndolter** [comment]
Re-tested with the iPhone 11 (ios 15.4) and the following issues still exist:
1. The focus is not on the GOC logo
2. Hidden content on top of the page continues to receive focus (e.g. H2 'You are here' near the breadcrumb, Primary tabs H2 where there is focus on both the Dashboard and Edit options together under Dashboard)
3. Items in the table continue to be announced by the screen reader but are not fully visible

**Comment by echouin** [comment]
I believe that this is not an issue in Prod. RSA CMS users are not recommended to use a mobile phone to add or modify content through the CMS.
